apache-rat-tasks/src/main/java/org/apache/rat/anttasks/All.java

```diff
@@ -27,6 +28,7 @@
  * @deprecated use configuration file.
  */
 @Deprecated // since 0.17
+@DeprecationReporter.Info(since = "0.17", forRemoval = true, use = "Configuration file and <config> element")// since 0.17
```

Comment:
Should this be <configuration> element here? Or is that only valid for the Maven plugin? Just to be correct in the help output ....

Reply:
This should be changed among many files in this PR once it's clear which element name is the correct one. I was unable to find any ANT examples that used a config, thus I'll keep my comment as a reminder here :)